### PR TITLE
feat: replace sort icon

### PR DESCRIPTION
**Pull request by YossiSaadi**
https://monday.monday.com/boards/3532714909/pulses/5070026459

### CODE REVIEW

File: src/Icons/Sort.svg [hunk 1]
```diff
@@ -1,3 +1,4 @@
-<svg width="20" height="20" viewBox="0 0 20 20" fill="none" xmlns="http://www.w3.org/2000/svg">
-<path fill-rule="evenodd" clip-rule="evenodd" d="M13.9697 7.53033C14.2626 7.82322 14.7374 7.82322 15.0303 7.53033C15.3232 7.23744 15.3232 6.76256 15.0303 6.46967L11.0303 2.46967C10.7374 2.17678 10.2626 2.17678 9.96967 2.46967L5.96967 6.46967C5.67678 6.76256 5.67678 7.23744 5.96967 7.53033C6.26256 7.82322 6.73744 7.82322 7.03033 7.53033L10.5 4.06066L13.9697 7.53033ZM15.0303 13.3875C15.3232 13.0946 15.3232 12.6197 15.0303 12.3268C14.7374 12.0339 14.2626 12.0339 13.9697 12.3268L10.5 15.7965L7.03033 12.3268C6.73744 12.0339 6.26256 12.0339 5.96967 12.3268C5.67678 12.6197 5.67678 13.0946 5.96967 13.3875L9.96967 17.3875C10.2626 17.6804 10.7374 17.6804 11.0303 17.3875L15.0303 13.3875Z" fill="currentColor"/>
+<svg width="21" height="21" viewBox="0 0 21 21" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (SergeyRoyt) [inline src/Icons/Sort.svg:1]: width, height and viewBox sizes should be 20

File: src/Icons/Sort.svg [hunk 2]
```diff
@@ -1,3 +1,4 @@
...
+<path d="M7.13869 2.75741C7.34727 2.75741 7.53593 2.84277 7.67174 2.98049L10.0716 5.38342C10.3641 5.67631 10.3641 6.15118 10.0716 6.44408C9.7791 6.73697 9.30483 6.73697 9.0123 6.44408L7.88775 5.3181V17.0578C7.88775 17.472 7.55238 17.8078 7.13869 17.8078C6.725 17.8078 6.38964 17.472 6.38964 17.0578V5.31805L5.26504 6.44408C4.97252 6.73697 4.49824 6.73697 4.20572 6.44408C3.9132 6.15118 3.9132 5.67631 4.20572 5.38342L6.60901 2.97708C6.62359 2.96249 6.63871 2.94855 6.65432 2.9353C6.78492 2.82434 6.954 2.75741 7.13869 2.75741Z" fill="#676879"/>
+<path d="M14.4434 17.8075C14.652 17.8075 14.8406 17.7222 14.9764 17.5844L17.3763 15.1815C17.6688 14.8886 17.6688 14.4138 17.3763 14.1209C17.0838 13.828 16.6095 13.828 16.317 14.1209L15.1924 15.2468V3.50712C15.1924 3.09291 14.8571 2.75712 14.4434 2.75712C14.0297 2.75712 13.6943 3.09291 13.6943 3.50712V15.2469L12.5697 14.1209C12.2772 13.828 11.8029 13.828 11.5104 14.1209C11.2179 14.4138 11.2179 14.8886 11.5104 15.1815L13.9137 17.5879C13.9283 17.6025 13.9434 17.6164 13.959 17.6296C14.0896 17.7406 14.2587 17.8075 14.4434 17.8075Z" fill="#676879"/>
```
Comment (SergeyRoyt) [inline src/Icons/Sort.svg:3]: Fill colors should be equal to `currentColor`, so icon would be able to inherit the color from the parent

**Review by SergeyRoyt — CHANGES_REQUESTED** [review]
Icon sizes and color should be fixed

**Comment by YossiSaadi** [comment]
> Icon sizes and color should be fixed

oh damn, my bad, I did it on Atmosphere and forgot about it here also
thanks for noticing 🙏🏼

**Review by SergeyRoyt — APPROVED** [review]

**Pull request merged**